### ISSUE TITLE
Bullet icon in bulleted-list item not visible in Windows high contrast mode

### ISSUE THREAD

**Issue by pharingee** [bbc/simorgh#4944]
**Describe the bug**
The bullet icon in psammead-bulleted-list is not visible in high contrast mode in Windows.

**To Reproduce**

1. Using a Windows machine switch to High Contrast mode.
2. Go to https://www.test.bbc.com/pidgin/23248703 

**Expected behavior**
The bullet icon should be visible in high contrast mode

**Screenshots**
![highcontrast2 (1)](https://user-images.githubusercontent.com/22519682/70989775-1ebf1080-20bc-11ea-975e-bf60bac6a24a.png)

![highcontrast1 (2)](https://user-images.githubusercontent.com/22519682/70989888-5a59da80-20bc-11ea-9162-8d5b5e3f0a7d.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by ryanmccombe** [comment]
Agreed to fix this for launch

May require some investigation, but likely to be a simple CSS addition

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-ms-high-contrast

**Comment by ryanmccombe** [comment]
Investigated this - not a simple CSS fix unfortunately.

Fixing this would require a rethink of how we have implemented these bulleted lists.

Bulleted lists use background images for the bullets - this is going to be problematic when it comes to high contrast modes.  They tend to be not that compatible with background images - can't find any official docs on this, but plenty of credible resources claiming it - this seems to be the most authoritive:

https://engineering.khanacademy.org/posts/windows-high-contrast-mode.htm

> Background colors and background images are removed.

> **Key takeaway**
> Only use background-color and background-image CSS for decorative styling. Windows High Contrast Mode considers them unnecessary to understand the page.

**Comment by gavinspence** [comment]
Won't address this issue as it will require a re-implementation of bulleted lists.
